### DISCUSSION TITLE
How to just run the Quarkus Dev once and exist the command line applicatipon

### DISCUSSION THREAD

**Discussion by yadaom** [quarkusio/quarkus#24155] [Q&A]
I am building an Quarkus command line application.
I am running Quarkus application with `./gradlew quarkusDev`.

But problem is that running the above command doesn't exists. It leaves the interactive mode on.
I want to run launch the application and exist without any interactive mode.
Because it is a command line application, I just want it to exist and run once only.

How todo it???

Please help!

**Comment by maxandersen** [comment]
java -jar target/quarkus-app/quarkus-run.jar

**Comment by yadaom** [comment]
Hello @maxandersen 

This won't compile the code.
I want to use the `gradle` to run compile and run in dev mode but just exist after the run.
Right now, `./gradlew quarkusDev` waits for the code to change and recompiles itself. 
I don't want that, I want to run and exist that's it.
